modules/statistics/server/jobs/daily-statistics.server.job.js

```diff
@@ -230,6 +230,41 @@ module.exports = function (job, agendaDone) {
           done,
         );
       },
+
+      // Get number of spoken languages
+      function (done) {
+        statistics.getUserLanguagesCount(40, function (err, languageCounts) {
+          if (err) {
+            log(
+              'error',
+              'Daily statistics: failed fetching spoken languages counts.',
+              err,
+            );
+            return done();
+          }
+
+          // Write numbers to stats
+          async.eachOfSeries(
```

Comment:
We run through each language and send stats to InfluxDB separately for each.

Reply:
Seems a bit inefficient as you can send many stats at once to influxdb, maybe we don't have that ability exposed right now though.

Not blocking.